Depricate example (#4)
DevExpress-Examples/blazor-dxtabs-add-remove-button: README.md

<!-- default badges list -->
[![](https://img.shields.io/badge/Open_in_DevExpress_Support_Center-FF7200?style=flat-square&logo=DevExpress&logoColor=white)](https://supportcenter.devexpress.com/ticket/details/T1108973)
[![](https://img.shields.io/badge/📖_How_to_use_DevExpress_Examples-e9f6fc?style=flat-square)](https://docs.devexpress.com/GeneralInformation/403183)
[![](https://img.shields.io/badge/💬_Leave_Feedback-feecdd?style=flat-square)](#does-this-example-address-your-development-requirementsobjectives)
<!-- default badges end -->
⛔ DEPRECATED. This example was deprecated.

> Beginning with v24.2.1 and onward, the [DxTabs](https://docs.devexpress.com/Blazor/DevExpress.Blazor.DxTabs) component implements tab Close buttons. Set the [ShowCloseButton](https://docs.devexpress.com/Blazor/DevExpress.Blazor.Base.DxTabBase.ShowCloseButton) property to true and handle the [TabClosing](https://docs.devexpress.com/Blazor/DevExpress.Blazor.DxTabs.TabClosing) event to close tabs. You can try this functionality in the [Tabs - Scroll Mode](https://demos.devexpress.com/blazor/Tabs#ScrollMode) demo.

Reviewer: ```suggestion
Starting from v24.2.1, the [DxTabs](https://docs.devexpress.com/Blazor/DevExpress.Blazor.DxTabs) component implements tab Close buttons. Set the [ShowCloseButton](https://docs.devexpress.com/Blazor/DevExpress.Blazor.Base.DxTabBase.ShowCloseButton) property to true and handle the [TabClosing](https://docs.devexpress.com/Blazor/DevExpress.Blazor.DxTabs.TabClosing) event to close tabs. You can try this functionality in the [Tabs - Scroll Mode](https://demos.devexpress.com/blazor/Tabs#ScrollMode) demo.
```
